domenetjenester/brevbestiller/src/main/java/no/nav/foreldrepenger/melding/brevmapper/brev/ForeldrepengerAnnullertDokumentdataMapper.java

```diff
@@ -45,12 +46,19 @@ public ForeldrepengerAnnullertDokumentdata mapTilDokumentdata(DokumentFelles dok
         var fellesBuilder = opprettFellesBuilder(dokumentFelles, hendelse, behandling, erUtkast);
         fellesBuilder.medBrevDato(dokumentFelles.getDokumentDato() != null ? formaterDato(dokumentFelles.getDokumentDato(), behandling.getSpråkkode()) : null);
 
+        StartdatoUtsattDto startdatoUtsatt = domeneobjektProvider.hentStartdatoUtsatt(behandling);
+        boolean harSøktOmNyPeriode = startdatoUtsatt.nyStartdato() != null;
```

Comment:
Obs på PR i fpsak der du endrer nyStartdato til nyStartDato - da blir den serialisert slik

Reply:
Jeg endret motsatt - nyStartDato til nyStartdato, så det skal være likt på begge sider i de to PRene